javaparser-core/src/main/java/com/github/javaparser/ast/expr/MethodCallExpr.java

```diff
@@ -318,4 +319,17 @@ public ResolvedMethodDeclaration resolve() {
     public Optional<MethodCallExpr> toMethodCallExpr() {
         return Optional.of(this);
     }
+    
+    @Override
+    /**
+     * Returns the comment or an Optional.empty if there is no comment on this method call.
+     * In case of MethodCallExpr, Comment are setted in an the parent ExpressionStmt comment attribute. 
+     */
+    public Optional<Comment> getComment() {
```

Comment:
This is not logical: a comment is always set on the node where you want it. This problem should not be solved by making a single node behave inconsistently.

Reply:
Ok it's probably caused by 

    Statement stmt = StaticJavaParser.parseStatement("System.out.println(\"World!\");");
    stmt.setLineComment(lineComment);

Reply:
Ah, you mean that for something like a `MethodCallExpr`, which can end up in a `ExpressionStmt`, we have to consider two nodes when deciding whether it has a comment? So we might need to be able to query whether a node is a "pseudonode" which doesn't have any source code text of its own? I think the lex.pres.printer has some "ghost node" logic (or so) for that. But @ftomassetti can explain if he has some time :-)

Reply:
@jlerbsc - I still don't really get what you want to happen, but I guess this PR can be merged.

Reply:
Oh i completely wrong and this PR should not be merged. In the issue declared by @Implex1v, he add a comment to a statement (not a MethodCallExpr) so i don't really fix the issue.

Reply:
Well, it's merged already and it did fix something, I think? Otherwise I (or you) will make a revert PR.